### PR TITLE
created titles component

### PR DESCRIPTION
**Pull request by xleonx0x**
created titles component with dynamic sizing!!!!!!!!

### CODE REVIEW

File: frontend/src/components/Title.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    switch(level) {
+        case "small":
+            size = 1;
+            break;
+        case "medium":
+            size = 1.5;
+            break;
+        case "large":
+            size = 2;
+            break;
```
Comment (bengodw) [inline frontend/src/components/Title.tsx]: wow - these sizes are really dynamic

**Review by bengodw — APPROVED** [review]
this looks great Chris. congrats on the awesome pr

File: frontend/src/components/Title.tsx (hunk 1 above)
Comment (xleonx0x) [reply frontend/src/components/Title.tsx]: wow thanks for the awesome picture

**Comment by lachlanshoesmith** [comment]
closing because i believe all changes were added in navbar pr

**Pull request closed without merging**